root/etc/e-smith/templates/etc/ssh/sshd_config/45AllowGroups2Sshd

```diff
@@ -36,14 +38,8 @@
     $admin .= "\@$domain" if ($admin !~ '@');
     # Spaces are not accepted
     $admin = '"'.$admin.'"' if ($admin=~ m/ /g);
```

Comment:
This block can be refactored like 

https://github.com/NethServer/nethserver-openssh/pull/9/files#diff-9db83e803f8c9bb93a998e8e292e5d36R11-R29

Reply:
done